static_src/components/users.jsx

```diff
@@ -24,7 +24,14 @@ const ORG_MANAGER = 'org_manager';
 const SPACE_MANAGER = 'space_manager';
 const ORG_ENTITY = 'organization';
 const SPACE_ENTITY = 'space';
-const cfCliLink = 'https://docs.cloudfoundry.org/adminguide/cli-user-management.html#space-roles';
+const ORG_INVITE_HELP = 'Only an Org Manager can new invite users to this ' +
```

Comment:
there was this commit that scarred me after it took a while finding the problem lol https://github.com/18F/cg-dashboard/commit/77a78f381efa28b1ebc80090cf1270ac3271bc51